src/sas_bluesky/ncd_panda.py

```diff
@@ -283,20 +281,6 @@ def check_and_apply_panda_settings(panda: HDFPanda, panda_name: str) -> MsgGener
         )
 
 
-def check_tetramm():
-    """
-    Checks if the tetramm is connected and returns the tetramm device.
-    If the tetramm is not connected, it will raise an error.
-    """
-
-    try:
-        tetramm = return_connected_device("i22", "tetramm")
```

Comment:
There is no device named "tetramm" in i22, and I am pushing as hard as possible to prevent people re-inventing the Finder from GDA. Pass the devices you expect into the plan, you will really appreciate it when it's time to debug.